src/fragments/start/getting-started/react/auth.mdx

```diff
@@ -58,6 +59,6 @@ You can also customize this component to add or remove fields, update styling, o
 
 In addition to the `withAuthenticator` you can build custom authentication flows using the `Auth` class.
```

Comment:
I don't think this is accurate, I don't think we expose the Auth class to allow this. Please check with @ErikCH and @wlee221

Reply:
@joebuono I believe you mean the `Auth` js library class? We don't expose this, but the JS library does. Maybe you can make that more explicit? Also the link will need to be updated in the next paragraph to here -> https://aws-amplify.github.io/amplify-js/api/classes/authclass.html

Reply:
I would recommend also clarifying that they can use the Auth JS library, however it would be in replacement of the authenticator, except in the case of using overide function calls.